src/components/Editor/Footer.js

```diff
@@ -56,22 +55,34 @@ class SourceFooter extends PureComponent<Props, State> {
   constructor() {
     super();
 
-    this.state = { cursorPosition: { line: 1, column: 1 } };
+    this.state = { cursorPosition: { line: 0, column: 0 } };
   }
 
-  componentDidMount() {
-    const { editor } = this.props;
-    editor.codeMirror.on("cursorActivity", this.onCursorChange);
+  componentDidUpdate() {
+    const eventDoc = document.querySelector(".CodeMirror");
```

Comment:
Hey @fvsch. 

The reason for using this selection logic in 2 places is that I'm forced to pass `eventDoc` into the method and declare it as an Object as a way to opt-out of type checking. https://flow.org/en/docs/types/objects/#toc-object-type

Otherwise, I'm receiving the Flow error when I try to access the CodeMirror property:
"Cannot get `CodeMirror` property in `HtmlElement` or `null`"

Would you or @darkwing have any suggestions to deal with this?